FLAnimatedImage/include/FLAnimatedBuiltInWebPDataSource.h

```diff
@@ -0,0 +1,19 @@
+//
+//  FLAnimatedBuiltInWebPDataSource.h
+//  Facebook
+//
+//  Created by Ben Hiller.
+//  Copyright (c) 2014-2015 Facebook. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <FLAnimatedImageFrameDataSource.h>
+
+@class WebPImageDecoder;
+
+@interface FLAnimatedBuiltInWebPDataSource : NSObject <FLAnimatedImageFrameDataSource>
+
+- (instancetype)initWithDecoder:(WebPImageDecoder *)decoder;
```

Comment:
I think it can just pass Webp data and create a decoder internally unless there is a performance issue.
```objc
- (instancetype)initWithData:(NSData *)data;
```